### PR TITLE
Add gzip command

### PR DESCRIPTION
**Pull request by Nabiz**
Add gzip command class and tests.

### CODE REVIEW

File: moler/cmd/unix/gzip.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+"""
+Gzip command module.
+"""
+
+__author__ = 'Dawid Gwizdz'
+__copyright__ = 'Copyright (C) 2019, Nokia'
+__email__ = 'dawid.gwizdz@nokia.com'
+
+from moler.cmd.unix.genericunix import GenericUnixCommand
```
Comment (Ernold11) [inline moler/cmd/unix/gzip.py]: please fix imports order:
import,
from ... import ...

File: moler/cmd/unix/gzip.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+    def _asks_to_overwrite(self, line):
+        """
+        Parse line containing overwriting warning.
+        :param line: Line from device.
+        :return: Nothing but raises ParsingDone if regex matches.
+        """
+        if self._regex_helper.search_compiled(self._re_overwrite, line):
+            if self.overwrite:
+                self.connection.sendline('y')
```
Comment (Ernold11) [inline moler/cmd/unix/gzip.py]: y will be send two times: first when prompt, second on temirnal echo

File: moler/cmd/unix/gzip.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            else:
+                self.connection.sendline('n')
```
Comment (Ernold11) [inline moler/cmd/unix/gzip.py]: same here

**Comment by marcin-usielski** [comment]
Please set Milestone.

**Pull request merged**